unittests/core/test_heartbeat.cpp

```diff
@@ -58,50 +58,53 @@ namespace {
 
     class HeartbeatAttachDetach : public ::testing::Test {
     public:
-        void SetUp () final {}
-        void TearDown () final {}
+        void SetUp () final {
```

Comment:
It's moot given my comments about but this could be moved to the HeartbeatAttachDetach ctor to avoid changing worker_ to a pointer.